.travis.yml

```diff
@@ -318,7 +318,7 @@ script:
 
   # compile and execute unit tests
   - mkdir -p build && cd build
-  - cmake .. ${CMAKE_OPTIONS} -DJSON_MultipleHeaders=${MULTIPLE_HEADERS} -DJSON_BuildTests=On -GNinja && cmake --build . --config Release
+  - cmake .. ${CMAKE_OPTIONS} -DJSON_ExplicitConversions=ON -DJSON_MultipleHeaders=${MULTIPLE_HEADERS} -DJSON_BuildTests=ON -GNinja && cmake --build . --config Release
```

Comment:
Why do you set the flag for all Travis tests? Wouldn't it be better to have it set for one test - just like what we do with disabled exceptions?